@aumuell I always get this error, independent with LLVM13, 12 and 11 when buiding the `ispc` dependency
```
 cd spack-stage-ispc-1.16.1-bj6yak4574yzkxi2ice6wia2fgnuhgaf/spack-src && ubuntu18.04-zen/gcc-7.5.0/m4-1.4.19-mfwutewkfi26kb6hwkxvrxwwrlw4wkno/bin/m4 -Ibuiltins -DLLVM_VERSION=LLVM_11_1 -DBUILD_OS=UNIX -DRUNTIME=32 
            builtins/target-sse4-i32x8.ll | ".../gcc-7.5.0/python-3.8.11-j47s6ghlntdvuntyspm7vcn7mupgc3a
            a/bin/python3.8" bitcode2cpp.py builtins/target-sse4-i32x8.ll --type=ispc-target --runtime=32 --os=UNIX --llvm_as .../gcc-7.5.0/llvm-11.1.0-3mxxrtsn5pnoajmyh3lxc4shk632yndr/bin/llvm-as > /home/kaindlb/home/spack/.stage/spack-stage-isp
            c-1.16.1-bj6yak4574yzkxi2ice6wia2fgnuhgaf/spack-build-bj6yak4/./builtins-target-sse4-i32x8-32bit-unix.cpp
     146    In file included from builtins/builtins-c-cpu.cpp:69:
     147    In file included from /usr/include/unistd.h:25:
  >> 148    /usr/include/features.h:424:12: fatal error: 'sys/cdefs.h' file not found
     149    #  include <sys/cdefs.h>
     150               ^~~~~~~~~~~~~
```

--- a/var/spack/repos/builtin/packages/ispc/package.py
+++ b/var/spack/repos/builtin/packages/ispc/package.py
@@ -50,6 +50,14 @@ class Ispc(CMakePackage):
     patch('fix-linking-against-llvm-10.patch', when='@1.13.0:1.13',
           sha256='d3ccf547d3ba59779fd375e10417a436318f2200d160febb9f830a26f0daefdc')
 
+    def patch(self):
+        with open("check-m32.c", "w") as f:
+            f.write('#include <sys/cdefs.h>')
+        try:
+            Executable(self.compiler.cc)('-m32', '-shared', 'check-m32.c', error=str)
+        except ProcessError:
+            filter_file('bit 32 64', 'bit 64', 'cmake/GenerateBuiltins.cmake')
+
     def cmake_args(self):
         args = []
         args.append('-DARM_ENABLED=FALSE')

